docs/flashbots-auction/searchers/advanced/rpc-endpoint.mdx

```diff
@@ -119,6 +119,86 @@ example response:
 }
 ```
 
+### eth_sendPrivateTransaction
+
+`eth_sendPrivateTransaction` is used to send a single transaction to Flashbots. Flashbots will attempt to send the transaction to miners for the next 25 blocks. See [Private Transactions](/flashbots-auction/searchers/advanced/private-transaction.mdx) for more info.
+
+This method has the following JSON-RPC format:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendPrivateTransaction",
+  "params": [{
+    tx,               // String, raw signed transaction
+    maxBlockNumber,   // Number, optional. Highest block number in which the transaction should be included.
+  }]
+}
+```
+
+example:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendPrivateTransaction",
+  "params": [{
+    "tx": "0x123abc...",
+    "maxBlockNumber": 14322777,
+  }]
+}
+```
+
+example response:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "0x45df1bc3de765927b053ec029fc9d15d6321945b23cac0614eb0b5e61f3a2f2a" // tx hash
+}
+```
+
+### eth_cancelPrivateTransaction
+
+The `eth_cancelPrivateTransaction` JSON-RPC method cancels private transactions. This method has the following JSON-RPC format:
```

Comment:
The `eth_cancelPrivateTransaction` JSON-RPC method cancels private transactions from being submitted for future blocks.

Reply:
:heavy_check_mark: fixed